fury/shaders/billboard_spheres_dec.frag

```diff
@@ -0,0 +1,3 @@
+
+    uniform mat4 MCDCMatrix;
+    uniform mat4 MCVCMatrix;
```

Comment:
this block can be removed. You do not use it, let's keep it simple

Reply:
This is needed for the depth component now.